apps/ccd/ccd-data-store-api/prod.yaml

```diff
@@ -19,7 +19,7 @@ spec:
         CCD_DOCUMENT_URL_PATTERN: ^https?://(((?:api-gateway\.dm\.reform\.hmcts\.net|dm-store-prod\.service\.core-compute-prod\.internal(?::\d+)?)\/documents\/[A-Za-z0-9-]+(?:\/binary)?)|(em-hrs-api-prod\.service\.core-compute-prod\.internal(?::\d+)?\/hearing-recordings\/[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}\/segments\/[0-9]+))
         CCD_S2S_AUTHORISED_SERVICES_CASE_USER_ROLES: aac_manage_case_assignment,fpl_case_service,finrem_case_orchestration,iac,divorce_frontend,civil_service,nfdiv_case_api,adoption_cos_api,adoption_web,prl_cos_api,probate_backend
         DATA_STORE_DB_HOST: ccd-data-store-api-postgres-db-v11-prod.postgres.database.azure.com
-        DATA_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_ps,probate_backend,divorce_ccd_submission,sscs,sscs_bulkscan,cmc,cmc_claim_store,cmc_claim_external_api,jui_webapp,pui_webapp,bulk_scan_orchestrator,fpl_case_service,iac,finrem_ccd_data_migrator,finrem_case_orchestration,employment_tribunals,ethos_repl_service,ccpay_bubble,ctsc_work_allocation,em_ccd_orchestrator,xui_webapp,bulk_scan_payment_processor,pcq_consolidation_service,em_npa_app,ecm_consumer,aac_manage_case_assignment,am_role_assignment_service,divorce_frontend,am_role_assignment_service,nfdiv_case_api,wa_task_configuration_api,civil_service,em_hrs_api,wa_task_monitor,ccd_case_document_am_api,wa_task_management_api,nfdiv_case_api,adoption_cos_api,adoption_web,prl_cos_api,hmc_cft_hearing_service,et_cos,et_msg_handler,ccd_next_hearing_date_updater,et_sya_api
+        DATA_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_ps,probate_backend,divorce_ccd_submission,sscs,sscs_bulkscan,cmc,cmc_claim_store,cmc_claim_external_api,jui_webapp,pui_webapp,bulk_scan_orchestrator,fpl_case_service,iac,finrem_ccd_data_migrator,finrem_case_orchestration,employment_tribunals,ethos_repl_service,ccpay_bubble,ctsc_work_allocation,em_ccd_orchestrator,xui_webapp,bulk_scan_payment_processor,pcq_consolidation_service,em_npa_app,ecm_consumer,aac_manage_case_assignment,am_role_assignment_service,divorce_frontend,am_role_assignment_service,nfdiv_case_api,wa_task_configuration_api,civil_service,em_hrs_api,wa_task_monitor,ccd_case_document_am_api,wa_task_management_api,nfdiv_case_api,adoption_cos_api,adoption_web,prl_cos_api,hmc_cft_hearing_service,et_cos,et_msg_handler,ccd_next_hearing_date_updater,et_sya_api,fis_hmc_api
```

Comment:
@Thomas-ForniCGI -  fis_hmc_api should be removed and conflicts has to be resolved.